.github/workflows/buildAndTest.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Build and Test'
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request: {}
+
+jobs:
+  build:
```

Comment:
What is the idea behind this job? It only runs `npm i`, but the test also does that 🤔

Reply:
I don't think that is necessary as the test job also installs the dependencies and will fail if that doesn't succeed.